CONTRIBUTING.md

```diff
@@ -1,109 +1,62 @@
-Contributing to Zeppelin
+Contributing to Open Zeppelin
 =======
 
-## Design Guidelines
+We really appreciate and value contributions to Open Zeppelin. Please take 5' to review the items listed below to make sure that your contributions are merged as soon as possible. 
 
-These are some global design goals in Zeppelin.
+## Contribution guidelines
 
-### D0 - Security in Depth
-We strive to provide secure, tested, audited code. To achieve this, we need to match intention with function. Thus, documentation, code clarity, community review and security discussions are fundamental.
+Smart contracts manage value and are highly vulnerable to errors and attacks. We have very strict guidelines, please make sure to review them: ["Contribution guidelines wiki entry"](https://github.com/OpenZeppelin/zeppelin-solidity/wiki/Contrbuting-guidelines).
 
-### D1 - Simple and Modular
-Simpler code means easier audits, and better understanding of what each component does. We look for small files, small contracts, and small functions. If you can separate a contract into two independent functionalities you should probably do it.
+## Branching
```

Comment:
I'd remove this branching section, and simply add a note in the next section to create the pull request against `developmnet` instead of `master`.